src/nvim/normal.c

```diff
@@ -2168,13 +2168,12 @@ static void op_function(oparg_T *oap)
  *
  * Return true if start_arrow() should be called for edit mode.
  */
-bool
```

Comment:
To make PRs easier to review, please try to avoid to make completely unrelated changes in a single PR (this seems to have nothing to do with `ops.c`)

Also, when you reformat a signature (which is ok if you do a related change to the function), please reformat to doxygen style, like
```
/// Yanks the text between "oap->start" and "oap->end" into a yank register.
/// If we are to append (uppercase register), we first yank into a new yank
/// register and then concatenate the old and the new one.
///
/// @param oap operator arguments
/// @param message show message when more than `&report` lines are yanked.
/// @returns whether the operation register was writable.
bool op_yank(oparg_T *oap, bool message)
```
i e param comments are before, and then the signature itself typically fits on one line.